File: Pythonleetcode/Leetcode168.py
class Solution(object):
    def convertToTitle(self, n):
        ans=""
        while n>0:
            n-=1
            remainder=n%26
            ans = ans + chr(remainder + ord("A"))
            n = n // 26

        return ans[::-1]


print(Solution().convertToTitle(28))
print(Solution().convertToTitle(27))
print(Solution().convertToTitle(26))
print(Solution().convertToTitle(26*26*26))

File: CppPractice/Project1/Project1/Leetcode168.cpp

#include <vector>

#include <iostream>

#include <stdlib.h>

#include <time.h>

#include <sstream>

#include <algorithm>

#include <map>

#include <bitset>

using namespace std;


class Solution {
public:
    string convertToTitle(int columnNumber) {
        string ans;
        while (columnNumber>0)
        {
            columnNumber--;
            ans += 'A' + (columnNumber % 26);
            columnNumber /= 26;
        }
        reverse(ans.begin(), ans.end());
        return ans;

    }
};